doc/configuration.md

```diff
@@ -277,6 +277,20 @@ gid         | int       | 0                 | no        | GID of the device owne
 mode        | int       | 0660              | no        | Mode of the device in the container
 required    | boolean   | false             | no        | Whether or not this device is required to start the container. (The default is no, and all devices are hot-pluggable.)
 
+### Type: gpu
+GPU device entries simply make the requested gpu device appear in the
+container.
+
+The following properties exist:
+
+Key         | Type      | Default           | Required  | Description
+:--         | :--       | :--               | :--       | :--
+vendorid    | string    | -                 | no        | The vendor id of the GPU device.
```

Comment:
Sorry, I meant "in addition" to the "id" field as they don't have the same meaning.

id is a card id, so 0 for the first card, 1 for the second, ... that matches the DRI or NVidia card id.

productid refers to the PCI productid

So productid=1234 may match more than one card if you have more than one card of the same model, vendor=abcd  & id=1 will only ever match a single card.